server/src/main/java/fr/uga/l3miage/tp1/exo2/models/SessionEntity.java

```diff
@@ -0,0 +1,16 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Entity
+@Table
+public class SessionEntity {
+    @Id
+    private UUID idSession ;
+    private String lastCommand ;
+    private String currentDir ;
+    private boolean lock ;
+    @OneToOne(mappedBy= "session")
+    private UserEntity user ;
```

Comment:
Ta relation est à l'envers, moi je veux que ce soit l'id de user dans la table session, donc c'est celui qui porte le `mappedBy` qui est propriétaire de la relation, et donc c'est lui qui donne son id. 

la dans ton cas, c'est session qui donne son id à User

```suggestion
    @OneToOne
    private UserEntity user ;
```